packages/mongodb-rag-core/src/contentStore/MongoDbEmbeddedContentStore.ts

```diff
@@ -58,6 +62,27 @@ export type MongoDbEmbeddedContentStore = EmbeddedContentStore &
     init(): Promise<void>;
   };
 
+function makeMatchQuery({ sourceNames, chunkAlgoHash }: GetSourcesMatchParams) {
+  let operator = "";
+  if (chunkAlgoHash.operation === "equals") {
+    operator = "$eq";
+  }
+  if (chunkAlgoHash.operation === "notEquals") {
+    operator = "$ne";
+  }
```

Comment:
```suggestion
  const operator = chunkAlgoHash.operation === "equals" ? "$eq" : "$ne";
```
Could simplify to a one-liner here since there are only two options